lib/contact_manager.rb

```diff
@@ -1,7 +1,7 @@
 class ContactManager
   require 'manage_phonebook'
 
-  attr_reader :options_menu, :phonebook
+  attr_reader :options_menu, :phonebook, :email, :phone
 
   @@manage_phonebook = ManagePhonebook.new()
```

Comment:
Could this live in `initialize` instead?

```ruby
def initialize
  @manage_phonebook = ManagePhonebook.new
end
```

I think generally you want to stay away from class variables (rather than instance variables) unless you really need them https://stackoverflow.com/questions/5890118/what-does-variable-mean-in-ruby

Reply:
changed from class variable to instance variable